### DISCUSSION TITLE
Why does `SPC-v` inserts `v` instead of returning `C-c v is undefined` ?

### DISCUSSION THREAD

**Discussion by JohnWick95** [meow-edit/meow#400] [General]
Installed from melpa and using https://github.com/meow-edit/meow/blob/master/KEYBINDING_QWERTY.org

**Comment by eshrh** [comment]
Discussed in https://github.com/meow-edit/meow/issues/196 (basically this behavior is better for the motion state)

You can set `meow-keypad-self-insert-undefined` to `nil` to get that behavior.

Also, we will currently message "v is undefined" in this case because if `C-c v` is undefined, we then check the binding on `v`.
